lib/broccoli/default-packager.js

```diff
@@ -676,6 +676,7 @@ module.exports = class DefaultPackager {
       let src = new Funnel(tree, {
         srcDir: 'src',
         destDir: 'src',
+        exclude: ['**/*-test.js'],
```

Comment:
Can we make the exclude only if MU is enabled?

Reply:
@rwjblue It is already done because this funnel is only executed if MU is enabled:

```
processSrc(tree) {
    if (this._cachedProcessedSrc === null && this.isModuleUnificationEnabled) {
      let src = new Funnel(tree, {
        srcDir: 'src',
        destDir: 'src',
        exclude: ['**/*-test.js'],
        annotation: 'Module Unification Src',
      });
```

I don't think it is needed to change the `exclude` behaviour, let me know otherwise.